documentation/src/main/asciidoc/topics/ref_xsite_jgroups_relay_custom.adoc

```diff
@@ -0,0 +1,34 @@
+[id='ref_relay_custom-{context}']
+= Custom JGroups RELAY2 Stacks
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<jgroups>
+   <stack-file name='relay-global' path='jgroups-relay.xml'/> <1>
+   <stack name="xsite" extends="udp">
+      <relay.RELAY2 site="LON" xmlns="urn:org:jgroups"
+                    max_site_masters='10'
+                    can_become_site_master='true'/> <2>
```

Comment:
`<2>` is in the wrong line. should be above.
Also, I'm not sure if single-quote `'` are allowed in the xml...

Reply:
nice catch. thanks.